CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ https://github.com/capistrano/capistrano/compare/v3.5.0...HEAD
   * Make path to git wrapper script configurable (@thickpaddy)
   * Change git wrapper path to work better with multiple users (@thickpaddy)
   * Make name of current directory configurable via configuration variable `:current_directory` (@websi)
+  * Updating the variables in the VariblesDoctor WHITELIST. (@shanesaww)
```

Comment:
You've explained what was changed, but not why. I think this would be better:

> `doctor` no longer erroneously warns that `:git_strategy` and other SCM options are "unrecognized"